Gemfile.lock

```diff
@@ -215,6 +217,7 @@ DEPENDENCIES
   rake
   retriable (~> 3.1)
   rspec
+  rubocop-rake
```

Comment:
This is unlikely to run by default, it'll probably need configuring in the ca style repo proper. Or patched in to our config in this repo inside the `.rubocop.yml` configuration file

NB: Disclaimer (I've never used this extension, so I could be wrong!)

Reply:
There's a `require: rubocop-rake` in the .rubocop.yml if you missed it. Caught a couple little things and as I was adding more to the rakefile it felt helpful.